src/text/text.rs

```diff
@@ -225,6 +225,19 @@ where
     }
 }
 
+impl std::fmt::Display for Text<'_> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let vec_len = self.lines.len();
+        for (i, line) in self.lines.iter().enumerate() {
+            write!(f, "{line}")?;
+            if (i + 1) != vec_len {
+                writeln!(f)?
+            }
+        }
```

Comment:
This can be simplified to
```suggestion
        for line in &self.lines {
            writeln!(f, "{line}")?;
        }
```

This will add a trailing `\n`, but is it a problem really?